internal/app/frontend/frontend.go

```diff
@@ -15,12 +15,33 @@
 package frontend
 
 import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
 	"google.golang.org/grpc"
 	"open-match.dev/open-match/internal/appmain"
 	"open-match.dev/open-match/internal/statestore"
+	"open-match.dev/open-match/internal/telemetry"
 	"open-match.dev/open-match/pkg/pb"
 )
 
+var (
+	totalBytesPerTicket   = stats.Int64("openmatch.dev/total_bytes_per_ticket", "Total bytes per ticket", stats.UnitBytes)
+	searchFieldsPerTicket = stats.Int64("openmatch.dev/searchfields_per_ticket", "Searchfields per ticket", stats.UnitDimensionless)
+
+	totalBytesPerTicketView = &view.View{
+		Measure:     totalBytesPerTicket,
+		Name:        "openmatch.dev/total_bytes_per_ticket",
```

Comment:
Why openmatch.dev, and not open-match.dev?

Reply:
I glanced through a bunch of examples but rarely see using `-` in the view name so I chose not to. I don't have a strong opinion here as long as the naming is consistent.

Reply:
open-match.dev is more consistently used, so I'd prefer that.